calico-enterprise/reference/resources/webhook.mdx

```diff
@@ -0,0 +1,151 @@
+
+### SecurityEventWebhookConfigVar
+
+| Field        | Description                                                                | Schema                                                                      | Required                            |
+| ------------ | -------------------------------------------------------------------------- | --------------------------------------------------------------------------- | ----------------------------------- |
+| name         | Configuration variable name.                                               | string                                                                      | yes                                 |
+| value        | Direct value for the variable.                                             | string                                                                      | yes if `valueFrom` is not specifed  |
+| valueFrom    | Value defined either in a Kubernetes ConfigMap or in a Kubernetesa Secret. | [SecurityEventWebhookConfigVarSource](#securityeventwebhookconfigvarsource) | yes if `value` is not specified     |
+
+### SecurityEventWebhookConfigVarSource
+
+| Field            | Description                       | Schema                                                                                                       | Required                                  |
+| ---------------- | --------------------------------- | ------------------------------------------------------------------------------------------------------------ | ----------------------------------------- |
+| configMapKeyRef  | Kubernetes ConfigMap reference.   | `ConfigMapKeySelector` (referenced ConfigMap key should exist in the `tigera-intrusion-detection` namespace) | yes if `secretKeyRef` is not specified    |
+| secretKeyRef     | Kubernetes Secret reference.      | `SecretKeySelector` (referenced Secret key should exist in the `tigera-intrusion-detection` namespace)       | yes if `configMapKeyRef` is not specifed  |
+
+### Status
+
+Field `status` reflects the health of a webhook. It is a list of [Kubernetes Conditions](https://pkg.go.dev/k8s.io/apimachinery@v0.23.0/pkg/apis/meta/v1#Condition).
+
+## Query
+
+SecurityEventWebhooks use a domain-specific query language to select which records
+from the data set should trigger the HTTP request.
+
+The query language is composed of any number of selectors, combined
+with boolean expressions (`AND`, `OR`, and `NOT`), set expressions
+(`IN` and `NOTIN`) and bracketed subexpressions. These are translated
+by {{prodname}} to Elastic DSL queries that are executed on the backend.
+
+Set expressions support wildcard operators asterisk (`*`) and question mark (`?`).
+The asterisk sign matches zero or more characters and the question mark matches a single character.
+Set values can be embedded into the query string or reference the values
+in the global alert substitution list.
```

Comment:
The "global alert substitution list" makes no sense in the context of this page.

Reply:
Removed now. Thanks @gantony!